### PR TITLE
[SYSDM] Fix french translation and layout

### PR DESCRIPTION
**Pull request by KRosUser**
## Before

image

## After

image

### CODE REVIEW

**Comment by binarymaster** [comment]
Cc @reactos/lang-french

**Review by zefklop — APPROVED** [review]

File: dll/cpl/sysdm/lang/fr-FR.rc [hunk 1]
```diff
@@ -367,7 +367,7 @@ BEGIN
     IDS_HWPROFILE_ALREADY_IN_USE "Le nom de profil est déjà utilisé."
     IDS_HWPROFILE_PROFILE "Profil"
     IDS_HWPROFILE_WARNING "Attention"
-    IDS_ENVIRONMENT_WARNING "Any changes that have been done will be discarded and the variable's value will be edited as text."
-    IDS_ENVIRONMENT_WARNING_TITLE "System Properties"
-    IDS_FILE_BROWSE_FILTER "All Files (*.*)\0*.*\0"
+    IDS_ENVIRONMENT_WARNING "Tous les changements effectués seront abandonnés et la valeur de la variable sera éditée en texte."
+    IDS_ENVIRONMENT_WARNING_TITLE "Propriétés système"
+    IDS_FILE_BROWSE_FILTER "Tous fichiers (*.*)\0*.*\0"
```
Comment (HBelusca) [inline dll/cpl/sysdm/lang/fr-FR.rc:372]:
```suggestion
    IDS_FILE_BROWSE_FILTER "Tous les fichiers (*.*)\0*.*\0"
```

File: dll/cpl/sysdm/lang/fr-FR.rc [hunk 2]
```diff
@@ -367,7 +367,7 @@ BEGIN
     IDS_HWPROFILE_ALREADY_IN_USE "Le nom de profil est déjà utilisé."
     IDS_HWPROFILE_PROFILE "Profil"
     IDS_HWPROFILE_WARNING "Attention"
-    IDS_ENVIRONMENT_WARNING "Any changes that have been done will be discarded and the variable's value will be edited as text."
-    IDS_ENVIRONMENT_WARNING_TITLE "System Properties"
-    IDS_FILE_BROWSE_FILTER "All Files (*.*)\0*.*\0"
+    IDS_ENVIRONMENT_WARNING "Tous les changements effectués seront abandonnés et la valeur de la variable sera éditée en texte."
```
Comment (HBelusca) [inline dll/cpl/sysdm/lang/fr-FR.rc:370]: "éditée en texte" , or: "éditée comme texte" ?

Comment (KRosUser) [reply dll/cpl/sysdm/lang/fr-FR.rc:370]: both looks good... what would you prefer ?

Comment (HBelusca) [reply dll/cpl/sysdm/lang/fr-FR.rc:370]: let's see what @zefklop and @hpoussin think about it :thinking:

**Pull request closed without merging**